pkg/tests/api/alerting/api_provisioning_test.go

```diff
@@ -809,3 +813,70 @@ func createTestRequest(method string, url string, user string, body string) *htt
 	}
 	return req
 }
+
+func TestIntegrationExportFileProvision(t *testing.T) {
+	testinfra.SQLiteIntegrationTest(t)
```

Comment:
Why do we need to stick only to Sqlite tests?

Reply:
Not sure, will double check that's necessary